notebooks/index.qmd

```diff
@@ -31,7 +31,7 @@ The **Quickstarts** examples are further divided into two sections, which you ca
 
 ## How to run
 
-Every notebook contains information about how to run it. Some can run on [mybinder](https://mybinder.org/) and all can run on the VEDA JupyterHub. See [VEDA Analytics JupyterHub Access](https://nasa-impact.github.io/veda-docs/veda-jh-access.html) for information about how to gain access.
+Every notebook contains information about how to run it. Some can run on [Binder](https://binder.openveda.cloud/) and all can run on the [VEDA JupyterHub](https://hub.openveda.cloud/). See [VEDA Analytics JupyterHub Access](https://nasa-impact.github.io/veda-docs/veda-jh-access.html) for information about how to gain access.
```

Comment:
should we pick a different term from Binder, that doesn't really explain what it is - but rather the technical tool that allows it. Maybe something like Trial or Test